posts/2022/09/hpy_sprint_status.md

```diff
@@ -0,0 +1,84 @@
+<!--
+.. title: HPy Sprint Status Update and Feedback Session
+.. slug: hpy_sprint_2022_report
+.. date: 2022-09-23 10:00:00 UTC
+.. author: fangerer
+.. tags:
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+As we announced in
+[this blog post](https://hpyproject.org/blog/posts/2022/07/dusseldorf-sprint-2022/),
+we are currently having our joint PyPy/HPy/GraalPy sprint in Dusseldorf.
+
+Yesterday, we did the HPy sprint report. Most notably, we also had several
+CPython core developers in the call.
+
+The sprint report was streamed on
+[Twitch](https://www.twitch.tv/pypyproject) and the recording is available here:
+<iframe width="560" height="315" src="https://www.youtube.com/embed/kl5w8uR6hGA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<!--TEASER_END-->
+
+
+## HPy Sprint Call with CPython Core Developers
+
+One of the main goals in the sprint was to write a PEP to officially propose
+HPy as an Python C extension API. We started to do so but before we actually
+submit this PEP, we wanted to have some feedback if that makes sense. So,
+we invited the Python core developers to our call and quite some of them
+followed the invitation.
+
+We therefore gave a short overview of HPy since not everyone might know it yet.
+We talked about the goals and non-goals of HPy and basically asked if Python
+core developers would be willing to support our project
```

Comment:
participate in HPy design discussions (and endorse HPy as the go-to API for Python extensions(?) -- not sure, does it follow from the discussion?). We discussed PSF endorsement and I think the conclusion was that we need to talk to PSF.